website_blog_excerpt_img/templates/assets.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Jairo Llopis
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+
+<data>
+
+    <template id="assets_frontend" inherit_id="website.assets_frontend">
+        <xpath expr=".">
+            <link rel="stylesheet"
+                  href="/website_blog_excerpt_img/static/src/css/website_blog_excerpt_img.scss"/>
```

Comment:
`css` --> `scss`

Reply:
It's already converted...

Reply:
@pedrobaeza I mean a directory

`/website_blog_excerpt_img/static/src/css/` --> `/website_blog_excerpt_img/static/src/scss/`

Reply:
The directory is the same for less, sass, scss or css: `css` as generic one for all.